easybuild/easyblocks/s/scipy.py

```diff
@@ -58,6 +62,19 @@ def configure_step(self):
             if self.toolchain.comp_family() in [toolchain.GCC, toolchain.CLANGGCC]:  # @UndefinedVariable
                 self.cfg.update('preinstallopts', "unset LDFLAGS && ")
 
+    def test_step(self):
+        """Run available scipy unit tests"""
+        if self.cfg['runtest']:
+            # Let's handle the output from the scipy test suite ourselves
+            testcmd_output, testcmd_exit_code = super(EB_scipy, self).test_step(return_testcmd_output=True)
+            scipy_testsuite_summary = parse_numpy_test_suite_output(testcmd_output)
+
+            if 'failed' in scipy_testsuite_summary or 'error' in scipy_testsuite_summary:
```

Comment:
Same here, `failed` vs `xfailed`?

Reply:
`numpy_testsuite_summary` is a dict, not a string